build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This is really nice, the script reduces our Travis runtime by like half since it only has to recursively run the tests once.